### ISSUE TITLE
Chapters - duplicate key value violates unique constraint

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-api#646]
```
2023-07-25 00:12:34.770 UTC [632309] ERROR:  duplicate key value violates unique constraint "mediaRef_index_episode_isOfficialChapter_startTime" 2023-07-25 00:12:34.770 UTC [632309] DETAIL:  Key ("episodeId", "isOfficialChapter", "startTime")=(2Lg1jiiKY, t, 11) already exists.
2023-07-25 00:12:34.770 UTC [632309] STATEMENT:  INSERT INTO "mediaRefs"("id", "endTime", "imageUrl", "isOfficialChapter", "isOfficialSoundBite", "isPublic", "linkUrl", "pastHourTotalUniquePageviews", "pastDayTotalUniquePageviews", "pastWeekTotalUniquePageviews", "pastMonthTotalUniquePageviews", "pastYearTotalUniquePageviews", "pastAllTimeTotalUniquePageviews", "startTime", "title", "createdAt", "updatedAt", "episodeId", "ownerId") VALUES ($1, DEFAULT, $2, $3, DEFAULT, $4, $5, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, DEFAULT, $6, $7, DEFAULT, DEFAULT, $8, $9) RETURNING "id", "int_id", "isOfficialChapter", "isOfficialSoundBite", "isPublic", "pastHourTotalUniquePageviews", "pastDayTotalUniquePageviews", "pastWeekTotalUniquePageviews", "pastMonthTotalUniquePageviews", "pastYearTotalUniquePageviews", "pastAllTimeTotalUniquePageviews", "startTime", "createdAt", "updatedAt"
```

**Comment by mitchdowney** [comment]
I'm not sure what scenarios this is happening in...but the error is related to [these lines of code](https://github.com/podverse/podverse-api/blob/develop/src/controllers/episode.ts#L522) and [this unique constraint](https://github.com/podverse/podverse-api/blob/develop/src/entities/mediaRef.ts#L24).

**Comment by mitchdowney** [comment]
This should be fixed in 4.14.12. We'll need to put the app into maintenance mode before running the alter table sql queries.
